modules/flowable-engine/src/test/java/org/flowable/engine/test/api/runtime/changestate/ChangeStateForMultiInstanceTest.java

```diff
@@ -58,8 +55,8 @@ protected void tearDown() {
     @Deployment(resources = "org/flowable/engine/test/api/multiInstanceSequential.bpmn20.xml")
     public void testSetCurrentActivityToSequentialMultiInstanceTask() {
         ProcessInstance processInstance = runtimeService.createProcessInstanceBuilder().processDefinitionKey("sequentialMultiInstance")
-                .variable("nrOfLoops", 3)
-                .start();
+                                                        .variable("nrOfLoops", 3)
```

Comment:
this formatting looks a little strange as the indentation is greater than project standards; are you using tabs instead of spaces?